charm/docs/tutorial/getting-started.md

```diff
@@ -34,7 +37,8 @@ juju add-model netbox-tutorial
 Deploy the NetBox charm, with all its mandatory requirements (PostgreSQL, Redis and S3).
```

Comment:
Move this line under the `Deploy the charms` section

Reply:
I have changed it a bit, as it was badly ordered..